packages/dd-trace/src/proxy.js

```diff
@@ -89,7 +86,7 @@ class Tracer extends NoopProxy {
       }
 
       if (config.remoteConfig.enabled && !config.isCiVisibility) {
-        const rc = remoteConfig.enable(config, this._modules.appsec)
+        const rc = require('./appsec/remote_config').enable(config, this._modules.appsec)
```

Comment:
This file is required in other places. Should the lazy loading happen in the module instead instead?

Reply:
would this const be assigned each time for normal runs besides cold start runs?